README.md

```diff
@@ -1 +1,2 @@
 <img src="/images/banner.jpg" alt="matthewctechnology">
+<a href="https:/matthewctechnology">matthewctechnology</a>
```

Comment:
why does this work?  The href here doesn't link as expected, so this is how it does work to href="https://github.com/matthewctechnology"